### PR TITLE
fix: camelCase request port options

### PR DESCRIPTION
**Pull request by mstahv**
Getting an exception without this change on Mac/Chrome/250 reader. Didn't inspect it more, but my guess is some chrome API is now more picky than it used to be. Don't yet have any eCards at my hand, but at least now the connection opens properly.

### CODE REVIEW

**Comment by mikaello** [comment]
Thank you both for showing interest and fixing this issue. As you pointed out, @rphlo, all of the options are camel-cased, in accordance with the [WCAG spec](https://wicg.github.io/serial/#serialoptions-dictionary).

I noticed that the [Google codelabs tutorial](https://codelabs.developers.google.com/codelabs/web-serial#3) uses lowercase for baudrate. I used this tutorial as one of my learning sources when I started this project, which might explain why I mistakenly used lowercase for all of them in this project.

**Comment by mikaello** [comment]
Do you want to give it a try to update the remaining options (and all places baud rate are used), @mstahv? Or do you want me to commit to this PR?

**Comment by mikaello** [comment]
And out of curiosity, have any of you used this library or have any ideas for usage?

My initial plan a few years ago, when this was created, was to make it very easy to manage timing in small races and to be able to use an Android phone or tablet instead of having to bring a laptop into the woods (it should, of course, work with a laptop as well). I never found the time to actually create the web app, just this library to communicate with the EMIT devices (which was the first step for such an application).

**Comment by mstahv** [comment]
I need to visit town to get an ecard on my hands to test/develop more. With the above the connection opened, but couldn't test forward. Plan was to make a tiny web UI for WMOC 2024 info to update the emit card number for competitors (without error prone typing of emit card number).

**Comment by mstahv** [comment]
Added another tiny change I was able to make without ecard, updated the serial port filters to 2024 compatible version.

**Comment by mikaello** [comment]
Cool! Nice that this can be useful to someone :-) I recently fixed what you also did in your latest commit, please rebase to get latest changes.

The tests should be fairly good, I have recorded quite a lot of actual real data from devices in https://github.com/mikaello/emit-punch-cards-communication/tree/main/src/mockdata to be able to test without the real equipment.

**Comment by mstahv** [comment]
Excellent, apparently I got you (and @rphlo ) interested again about this project 😎 If you change all parameters to came casing at once, feel free to drop this PR.

File: example/helper.ts [hunk 1]
```diff
@@ -34,10 +34,10 @@ const disconnectEventListener = async () => await disconnectEscan();
 export const connect250 = async () => {
   try {
     port250 = await navigator.serial.requestPort({
-      /* filters: [
+       /* filters: [
```
Comment (mikaello) [inline example/helper.ts:37]:
```suggestion
      /* filters: [
```

**Comment by mikaello** [comment]
Just for the history, it seems like this change in casing happened in Chrome 86, if this commit is legit: https://github.com/espruino/EspruinoTools/commit/fbc4a6c7e83e68ce07cbddbb27bdaff60b3a50ab

**Comment by mstahv** [comment]
Cool, I'm a contributor 😎 and quite confident I can make the ecard reading work once I get my hands on an ecard.

**Comment by mikaello** [comment]
Just a disclaimer: this library is not battle tested, as far as I know it hasn't been used by anyone (beside me), and I have myself just used it with the example in this repo to test that what I have been doing have worked. I guess there are edge cases and workflows I haven't considered that may not work.

But I'm very open for PRs if you find anything and have the time and energy to create any 😊

**Comment by mstahv** [comment]
Report: All works well 💪 Hacked together a Java API so can easily do anything I want. https://github.com/viritin/emit-reading-api

I'll share the actual infotool later, once the WMOC hassle is over. It is not pretty and probably as such not useful to anybody, but can maybe work as example and inspiration.
